docs/resources/samples/range-basics.md

```diff
@@ -1,281 +0,0 @@
----
```

Comment:
do we need a redirect for this file too?

Reply:
Good catch.